### DISCUSSION TITLE
Override base.html break theme

### DISCUSSION THREAD

**Discussion by zimocode** [squidfunk/mkdocs-material#2303] [Q&A]
the base.html source from : /material/base.html or /src/base.html .
the two base.html both break theme. didn't get the js an css files.
![image](https://user-images.githubusercontent.com/29518677/108184494-3f627a80-7146-11eb-9afe-17b00aa1747f.png)

AND, what is the right base.html file?

THANKS.

**Comment by squidfunk** [comment]
I'm afraid we can't help you with the given information. As a general note, when you override `base.html`, ensure that all `data-md-component` elements, as well as CSS and JavaScript files are still present, or the theme will not work correctly. It's almost always a better idea to use [block-level overrides](https://squidfunk.github.io/mkdocs-material/customization/#overriding-blocks).
